contracts/contracts/strategies/CurvePoolBoosterFactory.sol

```diff
@@ -0,0 +1,165 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import { Initializable } from "../utils/Initializable.sol";
+import { Strategizable } from "../governance/Strategizable.sol";
+import { CurvePoolBoosterPlain } from "./CurvePoolBoosterPlain.sol";
+import { ICreateX } from "../interfaces/ICreateX.sol";
+import { Initializable } from "../utils/Initializable.sol";
+
+/// @title CurvePoolBoosterFactory
+/// @author Origin Protocol
+/// @notice Factory contract to create CurvePoolBoosterPlain instances
+contract CurvePoolBoosterFactory is Initializable, Strategizable {
+
+    /// @notice Address of the CreateX contract
+    ICreateX public constant createX = ICreateX(0xba5Ed099633D3B313e4D5F7bdc1305d3c28ba5Ed);
```

Comment:
nit: i could be nice to have constants in capitale letters.

Reply:
tnx done here: https://github.com/OriginProtocol/origin-dollar/pull/2707/commits/f5ff05a9ca1ad3d89ee5f9c4233cdf7f1d56499b